Polish translations

wilkus27 opened the pull request:
Create translations of strings for Polish Psychology

jpslav reviewed (changes requested):
Please add an entry into the Unreleased section of the CHANGELOG.md.

Also, can you have some discussions (after this PR) with the OpenStax US styles team to think about how to tell Kitchen to use Polish vs English translations? cc @KarinaMendez2 @kswanson33 @LostMojo @brittany-johnson

wilkus27 commented:
@jpslav thank you! I added missing entry. 

About using translation, I just added this line for now  to pl-psychology `bake` file in Recipe repo:

![image](https://user-images.githubusercontent.com/40228854/115022979-70e59100-9ebe-11eb-93dd-2d07a2545691.png)

It's working, but I will bring this up on the call with the team, so we can find more elegant solution :slightly_smiling_face:
